ext/libclementine-tagreader/tagreader.cpp

```diff
@@ -130,6 +130,43 @@ const char* kASF_OriginalDate_ID = "WM/OriginalReleaseTime";
 const char* kASF_OriginalYear_ID = "WM/OriginalReleaseYear";
 }  // namespace
 
+// Helpers for GuessArtistAndTitle()
+static QString withoutExtension(const QString  s) {
```

Comment:
* Remove `static` and move inside the above anonymous namespace.
* Change arg to `const QString& s`
* Change name to `WithoutExtension`